package.json

```diff
@@ -30,16 +30,21 @@
     "ember-cli": "^2.4.2",
     "ember-cli-blueprint-test-helpers": "^0.12.0",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-internal-test-helpers": "^0.7.0",
+    "ember-cli-is-package-missing": "^1.0.0",
+    "ember-router-generator": "^1.2.1",
```

Comment:
You're right.

I can see that is-package-missing is required from https://github.com/emberjs/ember.js/blob/master/blueprints/helper-test/index.js

I'm wondering how blueprint tests can run without these. Is there a place where test only dependencies are specified? Or could it be that I'm running npm v2?